charts/gateway/values.yaml

```diff
@@ -623,7 +623,11 @@ otk:
     #   keyspace:
     # For Cassandra driverConfig is supported from GW 11.0. Either otk.database.cassandra.driverConfig or otk.database.properties should be provided
     #   driverConfig: {"localDataCenterName" : "DC1"}
-    
+
```

Comment:
please update production-values.yaml file

Reply:
this is updated